imap/cyr_dbtool.c

```diff
@@ -434,18 +433,22 @@ int main(int argc, char *argv[])
     } else if (!strcmp(action, "repack")) {
         r = cyrusdb_repack(db);
     } else if (!strcmp(action, "damage")) {
-        cyrusdb_store(db, "INVALID", 7, "CRASHME", 7, &tid);
-        assert(!tid);
+        cyrusdb_store(db, "INVALID", 7, "CRASHME", 7, tidp);
+        assert(!*tidp);
```

Comment:
`tidp` is initialized to `NULL`, so this could become a NULL pointer dereference